src/components/remote_control/src/commands/set_interior_vehicle_data_request.cc

```diff
@@ -61,6 +61,30 @@ std::vector<std::string> GetModuleReadOnlyParams(
   return module_ro_params;
 }
 
+const std::map<std::string, std::string> GetModuleDataToCapabilitiesMapping() {
+  std::map<std::string, std::string> maping;
```

Comment:
@LuxoftAKutsan 
'mapping' or even better 'capabilities'
I think its better to makes this static instead of new interface